content/patterns/disclosure/examples/disclosure-navigation-hybrid.html

```diff
@@ -20,183 +20,176 @@
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Transcripts</a>
+                    <a href="#mythical-page-content">成績証明書</a>
                   </li>
                 </ul>
               </li>
             </ul>
           </nav>
           <div id="mythical-page-content" class="disclosure-page-content" tabindex="-1" role="region" aria-labelledby="mythical-page-heading">
-            <h3 id="mythical-page-heading">Home</h3>
+            <h3 id="mythical-page-heading">ホーム</h3>
             <p>
-              Sample content section.
-              Activating a link above will update and navigate to this region.
+              サンプルコンテンツセクション。
+              上記のリンクを選択すると、この領域が更新されてナビゲートされます。
             </p>
           </div>
-          <footer class="sample-footer">Mythical University footer information</footer>
+          <footer class="sample-footer">架空大学のフッター情報</footer>
         </div>
         <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ol>
-          <li>Since this set of links and disclosure buttons provides navigation for the mythical university web site, the list that contains them is wrapped in a navigation landmark named <q>Mythical University</q>.</li>
+          <li>このリンクとディスクロージャーボタンのセットは架空大学ウェブサイトのナビゲーションを提供するため、それらを含むリストは<q>架空大学</q>と名付けられたナビゲーションランドマークでラップされています。</li>
           <li>
-            The semantics of the list structure communicate the hierarchy of the navigation system to assistive technology users.
-            The top-level list has three items where each item contains a top-level link and an associated disclosure button.
-            The set of links controlled by a disclosure button is in a list nested inside the list item that contains the button.
+            リスト構造のセマンティクスは、支援技術の利用者にナビゲーションシステムの階層を伝えます。トップレベルのリストは　3　つのアイテムをもつリストで、それぞれの項目はトップレベルリンク及びそれに関連するディスクロージャーボタンを含みます。ディスクロージャーボタンによって制御されるリンクのセットは、そのボタンを含むリストアイテム内にネストされたリストにあります。
           </li>
           <li>
-            If a dropdown is open and focus is inside the navigation region, pressing <kbd>Esc</kbd> will close the dropdown.
-            Moving focus out of the navigation region also closes an open dropdown.
+            ドロップダウンが開かれ、フォーカスがナビゲーション領域内にある場合、<kbd>Esc</kbd> を押すとドロップダウンが閉じます。ナビゲーション領域からフォーカスを外すことでも、開いているドロップダウンは閉じます。
           </li>
           <li>
-            Optional navigation keys (Arrow keys, <kbd>Home</kbd>, and <kbd>End</kbd>):
+            オプショナルなナビゲーションキー（矢印キー及び　<kbd>Home</kbd>、並びに　<kbd>End</kbd>）:
             <ol>
-              <li>Enabling the optional arrow key support prevents the default page scroll behavior when focus is contained within the navigation region.</li>
+              <li>オプショナルな矢印キーサポートを有効にすると、フォーカスがナビゲーション領域内に含まれる場合、デフォルトのページスクロール動作が防がれます。</li>
```

Comment:
原文では特に受動態で書かれているわけではないので、「ページスクロール動作が防がれます」は「ページスクロール動作を防ぎます」でよいかなと思います。

```suggestion
              <li>オプショナルな矢印キーサポートを有効にすると、フォーカスがナビゲーション領域内に含まれる場合、デフォルトのページスクロール動作を防ぎます。</li>
```

Reply:
「オプションの」はその通りですね。
受動態でないというのも、そうでした。
ありがとうございます！